services/onboarding-service/src/main/java/com/amazon/aws/partners/saasfactory/saasboost/OnboardingService.java

```diff
@@ -1835,19 +1835,9 @@ protected void handleUpdateInfrastructure(Map<String, Object> event, Context con
         Map<String, Object> services = (Map<String, Object>) appConfig.get("services");
 
         String domainName = (String) appConfig.getOrDefault("domainName", "");
```

Comment:
We can remove both domain name and hosted zone from this method. They aren't part of the CloudFormation stack anymore.